gulpfile.js

```diff
@@ -30,24 +31,24 @@ gulp.task('css', function () {
         .pipe(gulp.dest('./target/css'));
 });
 
+gulp.task('js', function(done) {
+    jspm.bundleSFX('src/js/app', './target/js/bundle.js', {
```

Comment:
Running the build via the API here as that seemed the nicest way to integrate with gulp. Not sure if it's preferable to just call out to `jspm bundle-sfx` as a shell task.